pkgs/by-name/no/nova-password/package.nix

```diff
@@ -5,17 +5,17 @@
 
 buildGoModule rec {
   pname = "nova-password";
-  version = "0.5.6";
+  version = "0.5.7";
 
   src = fetchFromGitHub {
     owner = "sapcc";
     repo = "nova-password";
     rev = "refs/tags/v${version}";
-    hash = "sha256-pBew5c+wlXwMLDjgwwdOSUyTxPGpa9AwbhZni8FfTsQ=";
+    hash = "sha256-tjjP+LmYaqpzmTk3tgFqZlG1KEeAkfI7RxzSm97jWVU=";
   };
 
   # The repo contains vendored dependencies
```

Comment:
```suggestion
```

Not anymore since https://github.com/sapcc/nova-password/commit/4e80400d2ec931b101cda4e5e885754e6e9022f5